### ISSUE TITLE
Rename refactor does not accept dashes

### ISSUE THREAD

**Issue by johnsoft** [VladRassokhin/intellij-hcl#112]
When renaming a resource using the refactor dialog, it won't accept a name with dashes. Terraform itself is fine with dashes in a resource name.

### Prerequisites
 - [x] Ensure you have latest version of plugin installed
 - [x] Search for possible issue duplicates

### Installation details
 - IDE version:
```
PyCharm 2017.2.3
Build #PY-172.3968.37, built on September 1, 2017
Licensed to Noah Jacobson
Subscription is active until December 8, 2017
JRE: 1.8.0_152-release-915-b11 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Windows 10 10.0
```

 - intellij-hcl plugin version 0.6.7
 - Terraform version: 0.10.7

### Expected Behavior

Should be able to rename

### Actual Behavior
Unable to rename

![](https://i.imgur.com/b9odO09.png)

### Steps to Reproduce

1. Add this to a `.tf` file:

```resource "aws_key_pair" "terraform-deploy" {}```

2. Open the rename dialog for `terraform-deploy`.

**Comment by VladRassokhin** [comment]
Would be released in 0.6.9

**Comment by VladRassokhin** [comment]
FYI 0.6.9 just released.
